2/contrib/s2i/run

```diff
@@ -103,6 +103,25 @@ if [ ! -e ${JENKINS_HOME}/configured ]; then
   cp -r /opt/openshift/configuration/* ${JENKINS_HOME}
   rm -rf /opt/openshift/configuration/*
 
+
+  # If the INSTALL_PLUGINS variable is populated, then attempt to install
+  # those plugins before copying them over to JENKINS_HOME
+  # The format of the INSTALL_PLUGINS variable is a comma-separated list
+  # of pluginId:pluginVersion strings
+  if [[ ! -z ${INSTALL_PLUGINS:-} ]]; then
+    echo "Installing additional plugins: ${INSTALL_PLUGINS} ..."
+
+    # Create a temporary file in the format of plugins.txt
+    plugins_file=$(mktemp)
+    IFS=',' read -ra plugins <<< "${INSTALL_PLUGINS}"
```

Comment:
why would the statement as it is leak IFS? It's only setting it for the read statement. I think that's cleaner than setting it and cleaning it up after the loop.